src/qml/Mysthea/Pages/MainMenu.qml

```diff
@@ -13,34 +13,17 @@ Page {
     property bool hasToolbarLine: false
 
     signal cardsReferenceClicked
-    signal gameSetupClicked
-    signal rulesbookClicked
-    signal loreClicked
-    signal languageClicked
-    signal creditsClicked
 
-    background: Image {
-        source: "qrc:/assets/images/home-bg.jpg"
-        fillMode: Image.PreserveAspectCrop
-        smooth: false
-
-        horizontalAlignment: Image.AlignHCenter
-        verticalAlignment: Image.AlignBottom
-
-        Rectangle {
-            anchors.fill: parent
-            color: Palette.mineShaft
-            opacity: 0.29
-        }
+    background: Rectangle {
+        anchors.fill: parent
+        color: Palette.mineShaft
+        opacity: 0.29
     }
 
     ColumnLayout {
         id: logoContainer
-        width: parent.width
-        anchors.leftMargin: 32
-        anchors.rightMargin: 32
-        anchors.top: parent.top
-        anchors.bottom: container.top
+        anchors.fill: parent
+        anchors.margins: 32
 
         Image {
```

Comment:
Maybe, in this case, it is better use a `Button` component that has yet implemented the clicked function without add the `MouseArea`.  
Since `Button` has `background` property, we can customize it and add the `Image` and the `Icon` as background content. 
[Button properties](https://doc.qt.io/qt-5/qml-qtquick-controls2-button-members.html)